src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
For what it's worth, the convention in Hapi is to capitalize anything you 'require'.  It makes it more clear which variables are from an external file, and I agree with @jbucaran that it looks nicer.